### DISCUSSION TITLE
HOW TO FIX THIS?

### DISCUSSION THREAD

**Discussion by ah-ha-admin** [termux/termux-app#3247] [Q&A] [answered]
![Screenshot_20230212-114209_Termux](https://user-images.githubusercontent.com/118419708/218324454-0c604c17-e135-4913-8541-120bfc1a3ac2.jpg)

IM USING A TABLET SAMSUNG RUNNING ON ANDROID 11

**Comment by ah-ha-admin** [comment]
@EdwardBetts @bumper314 @lotheac @jeansch

**Comment by Unknowncoder46** [comment]
> ![Screenshot_20230212-114209_Termux](https://user-images.githubusercontent.com/118419708/218324454-0c604c17-e135-4913-8541-120bfc1a3ac2.jpg)
> 
> IM USING A TABLET SAMSUNG RUNNING ON ANDROID 11

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-tasker#allow-external-apps-property-optional
